### ISSUE TITLE
Add download file button

### ISSUE THREAD

**Issue by ycaniou** [freeciv/freeciv-web#676]
At the moment, one can only load a saved file.
It would be nice to be able to download a saved file as well, to play locally some current game.
Cheers.

**Comment by rvid19** [comment]
Yes, it would be useful.

**Comment by rvid19** [comment]
Adding an upload button sounds cool, but it will surely cause some problems.

**Comment by ycaniou** [comment]
You're right, security issue at least, possibility to overload a server, bypass the one save by turn limitation (which can already be bypassed by saving, quiting, reloading, and saving, but a bit less straightforward)..
But download would be great.
